packages/enoki/src/react.tsx

```diff
@@ -1,24 +1,29 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import type { SuiClient } from '@mysten/sui/client';
 import { useStore } from '@nanostores/react';
 import type { ReactNode } from 'react';
-import { createContext, useContext, useEffect, useState } from 'react';
+import { createContext, useContext, useEffect, useMemo, useState } from 'react';
 
-import type { EnokiFlowConfig } from './EnokiFlow.js';
+import type { AuthProvider, EnokiFlowConfig } from './EnokiFlow.js';
 import { EnokiFlow } from './EnokiFlow.js';
+import type { EnokiWallet } from './wallet/index.js';
+import { registerEnokiWallets } from './wallet/index.js';
 
 const EnokiFlowContext = createContext<EnokiFlow | null>(null);
 
 export interface EnokiFlowProviderProps extends EnokiFlowConfig {
 	children: ReactNode;
 }
 
+/** @deprecated use EnokiWalletProvider instead  */
 export function EnokiFlowProvider({ children, ...config }: EnokiFlowProviderProps) {
 	const [enokiFlow] = useState(() => new EnokiFlow(config));
 	return <EnokiFlowContext.Provider value={enokiFlow}>{children}</EnokiFlowContext.Provider>;
 }
 
+/** @deprecated use EnokiFlowProvider and dapp-kit wallet hooks instead */
```

Comment:
```suggestion
/** @deprecated use EnokiWalletProvider and dapp-kit wallet hooks instead */
```

small typo correction for all of these